hopsworks-api/src/main/java/io/hops/hopsworks/api/project/RequestService.java

```diff
@@ -239,18 +243,26 @@ public Response requestJoin(RequestDTO requestDTO, @Context HttpServletRequest r
             getOwner().getLname() + ", \n\n"
             + user.getFname() + " " + user.getLname()
             + " wants to join a project you own. \n\n"
-            + "Project name: " + project.getName() + "\n"
-            + "Attached message: " + requestDTO.getMessageContent() + "\n"
-            + "After loging in to hopsworks go to : /project" + project.getId()
-            + " and go to members tab "
-            + "if you want to add this person as a member in your project. \n";
+            + "Project name: " + project.getName() + "\n";
+
+    if(requestDTO.getMessageContent() != null) {
+      msg += "Attached message: " + requestDTO.getMessageContent() + "\n";
+    }
+
+    msg += "After loging in to Hopsworks go to : /project" + project.getId()
+        + " and go to members tab "
+        + "if you want to add this person as a member in your project. \n";
 
     Users from = user;
     Users to = userFacade.findByEmail(project.getOwner().getEmail());
     String message = "Hi " + to.getFname() + "<br>"
             + "I would like to join a project you own. <br>"
-            + "Project name: " + project.getName() + "<br>"
-            + requestDTO.getMessageContent();
+            + "Project name: " + project.getName() + "<br>";
+
+    if(requestDTO.getMessageContent() != null) {
```

Comment:
@robzor92 should we check for empty message as well here? as in Strings.isNullorEmpty()

Reply:
done